src/Utils.ts

```diff
@@ -8,9 +8,13 @@ export class UVUtils {
         return filterXSS(html, {
             whiteList: {
                 a: ["href", "title", "target", "class"],
+                b: [],
                 br: [],
+                i: [],
                 img: ["src"],
-                span: []
+                p: [],
+                span: [],
+                u: []

```

Comment:
It does seem to make sense to add sub, sup and small (I don't think small is already there). I suggested including the u tag because we might use it in some of our legacy data for underlining, but I see that it was deprecated in HTML 4.01 and repurposed in HTML5, so that's probably why it was omitted from the Presentation 3 spec.

Reply:
Thanks for the feedback, the code has been updated.

Reply:
Thanks @Geoffsc